buildsystem/dependencies.gradle

```diff
@@ -39,7 +39,7 @@ ext.versions = [
         recyclerView                : '1.2.1',
         retrofit                    : '2.9.0',
         retrofitSerializerConverter : '0.8.0',
-        room                        : '2.3.0',
+        room                        : '2.4.0-alpha03',
```

Comment:
2.4.2 is actually stable and fixes this as well for M1 macs.  Can you update that, and then I can merge.

Room Releases: https://developer.android.com/jetpack/androidx/releases/room#2.4.2

Reply:
It looks like there is already an open PR to bump to 2.4.1 here: https://github.com/dropbox/Store/pull/385